graphql-server/migrations/1589049353064_alter_table_public_games_add_column_allow_card_skips/up.yaml

```diff
@@ -0,0 +1,6 @@
+- args:
+    cascade: false
+    read_only: false
+    sql: ALTER TABLE "public"."games" ADD COLUMN "allow_card_skips" boolean NOT NULL
+      DEFAULT true;
```

Comment:
Certainly open to chat about a more flexible data type here.

Reply:
I'm cool with boolean - what did you have in mind?

(Sometimes I use a timestamp w/zone, like `allow_card_skips_at` bc technically it holds more info. Seems over the top here.)

Reply:
Yeah no worries. My only other (after)thought was an integer if we thought we might expand the setting in the future (i.e. max number of skips or something).

Reply:
Ah yea, I thought of that as another column.. # of skips. But now i realize that could store all the settings

null - infinite skips
0 - no skips
1+ - # of skips allowed

we can add another column and migrate later or something